tests/conftest.py

```diff
@@ -89,6 +85,84 @@ def sat_data():
     return ds
 
 
+@pytest.fixture
+def valid_config_dict(model_minutes_kwargs):
+    cfg = {
+        "_target_": "pvnet.models.multimodal.multimodal.Model",
+        "embedding_dim": 16,
+        "include_sun": True,
+        "include_gsp_yield_history": True,
+        "forecast_minutes": 420,
+        "history_minutes": 180,
+
+        "output_quantiles": [0.1, 0.5, 0.9],
+
+        "output_network": {"_target_": "pvnet.models.multimodal.linear_networks.networks.ResFCNet2"},
+        "optimizer": {"_target_": "torch.optim.Adam", "lr": 0.001},
+
+        "sat_encoder": {
+            "_target_": "pvnet.models.multimodal.encoders.encoders3d.DefaultPVNet",
+            "in_channels": 11, "out_features": 128, "number_of_conv3d_layers": 6,
+            "conv3d_channels": 32, "image_size_pixels": 24,
+        },
+
+        "nwp_encoders_dict": {
+            "ukv": {
+                "_target_": "pvnet.models.multimodal.encoders.encoders3d.DefaultPVNet",
+                "in_channels": 11, "out_features": 128, "number_of_conv3d_layers": 6,
+                "conv3d_channels": 32, "image_size_pixels": 24,
+            },
+            "ecmwf": {
+                "_target_": "pvnet.models.multimodal.encoders.encoders3d.DefaultPVNet",
+                "in_channels": 12, "out_features": 128, "number_of_conv3d_layers": 6,
+                "conv3d_channels": 32, "image_size_pixels": 12,
+            },
+        },
+
+        "sat_history_minutes": 30,
+        "nwp_history_minutes": {
+            "ukv": 120,
+            "ecmwf": 120,
+        },
+        "nwp_forecast_minutes": {
+            "ukv": 480,
+            "ecmwf": 480,
+        },
+        "nwp_interval_minutes": {
+             "ukv": 60,
+             "ecmwf": 60,
+        },
+
+        "satellite_time_resolution_minutes": 5,
+        "gsp_time_resolution_minutes": 30,
+        "sun_time_resolution_minutes": 30,
+    }
+    return cfg
+
+
+def _convert_to_numpy_recursive(item):
```

Comment:
do we have this already, 

I think so, could be `from ocf_data_sampler.numpy_sample.collate import stack_np_samples_into_batch`